CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## 0.8.1
```

Comment:
This should be 0.9.0 because it's the addition of a new feature.
I'll update it after the merge.